geode-core/src/distributedTest/java/org/apache/geode/internal/cache/versions/TombstoneDUnitTest.java

```diff
@@ -117,35 +118,74 @@ public void testTombstoneGcMessagesBetweenPersistentAndNonPersistentRegion() {
     });
   }
 
+
   @Test
-  public void testGetOldestTombstoneTimeReplicate() {
+  public void testWhenAnOutOfRangeTimeStampIsSeenWeExpireItInReplicateTombstoneSweeper() {
     VM server1 = VM.getVM(0);
     VM server2 = VM.getVM(1);
+    final int FAR_INTO_THE_FUTURE = 1000000; // 1 million millis into the future
 
+    final int count = 10;
     server1.invoke(() -> {
-      createCacheAndRegion(REPLICATE_PERSISTENT);
-      region.put("K1", "V1");
-      region.put("K2", "V2");
+      createCacheAndRegion(RegionShortcut.REPLICATE_PERSISTENT);
+      for (int i = 0; i < count; i++) {
+        region.put("K" + i, "V" + i);
+      }
     });
 
-    server2.invoke(() -> createCacheAndRegion(REPLICATE));
+    server2.invoke(() -> createCacheAndRegion(RegionShortcut.REPLICATE));
```

Comment:
Can you explain a bit why the region shortcut on server2 is not the same as that of server1 `REPLICATE_PERSISTENT` ? I understand this is not your code change though.

Reply:
fixed that.